syntax/parse.go

```diff
@@ -90,9 +89,10 @@ sexpr  -> "${"
 
 ARROW  -> /{:>|=>|>>|orderby|order|where|sum|max|mean|median|min};
 IDENT  -> /{ \. | [$@A-Za-z_][0-9$@A-Za-z_]* };
+STRING -> /{ [$@A-Za-z_/][0-9$@A-Za-z_:.\/-]* };
```

Comment:
```suggestion
PKGPATH -> /{ { (?: \\. | [^\\}] )* \} };
```

Reply:
need to take the `.` out, there is an issue with this rule when trying to compile `{1,2,3,4} => .^2` looks like the . `^2` are separated and got issue with `^2` have no match rule.